eol-versions.js

```diff
@@ -0,0 +1,125 @@
+// Sync from https://raw.githubusercontent.com/nodejs/Release/master/schedule.json
+// These are the list of versions that might be affected by this module dependencies
+const versionMap = {
```

Comment:
Move to local `schedule.json` so that it's easy to maintain?

Reply:
This list shouldn't change anymore. At least for the next 4 years I'd say. All next EOL versions does support `@pkjs/nv` and we can lock it in the package-lock.json